modules/oe_webtools_cookie_consent/oe_webtools_cookie_consent.module

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Webtools cookie consent module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\media\IFrameMarkup;
+
+/**
+ * Implements hook_preprocess_HOOK() for media-oembed-iframe.html.twig template.
+ *
+ * According to the fact that from oEmbed service we receive
```

Comment:
```
Implements template_preprocess_media_oembed_iframe().

Because the oEmbed format comes with the actual HTML iframe, we have to change its "src" attribute value with the URL of the Cookie Consent service, passing the original URL to it.
```

Reply:
Corrected